### PR TITLE
Remove force resync feature

### PR DESCRIPTION
**Pull request by piotr-iohk**
# Issue Number

N/A

# Overview

- 05ae60a7f6dd8fb7b87b7cc07e41a4bc77d22f64
  remove integration tests
  
- e650619499adebb16ae798665b9e0f57d199c14f
  remove forceResync feature
  
- a479fad727fa9f12a31d78415c65f01e44bd1eef
  remove forceResync from swagger spec

# Comments

_All tests passed for me locally._

### CODE REVIEW

File: lib/core/src/Cardano/Wallet/Api/Server.hs [hunk 1]
```diff
@@ -979,44 +976,6 @@ getUTxOsStatistics ctx (ApiT wid) = do
...
 
-forceResyncWallet
-    :: forall ctx s t k.
-        ( ctx ~ ApiLayer s t k
-        , IsOurs s ChimericAccount
-        , IsOurs s Address
-        )
-    => ctx
-    -> ApiT WalletId
-    -> ApiNetworkTip
```
Comment (KtorZ) [inline lib/core/src/Cardano/Wallet/Api/Server.hs:990]: I wonder if this one (`ApiNetworkTip` is used for anything else or could also be removed.

**Review by KtorZ — APPROVED** [review]
:+1:

File: lib/core/src/Cardano/Wallet/Api/Server.hs (hunk 1 above)
Comment (piotr-iohk) [reply lib/core/src/Cardano/Wallet/Api/Server.hs:990]: I think it is -> https://github.com/input-output-hk/cardano-wallet/blob/master/lib/core/src/Cardano/Wallet/Api/Types.hs#L571

**Comment by KtorZ** [comment]
bors retry

**Pull request merged**